src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,13 @@
 package mate.academy.service;
 
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+    private static final UserService userService = new UserService();
```

Comment:
```suggestion
    private final UserService userService = new UserService();
```

Reply:
I understand that we working with objects that is why we need to stay out of static? Or there is some wiser explanation? Thanks)